presto-hive/src/main/java/com/facebook/presto/hive/metastore/Table.java

```diff
@@ -106,6 +106,11 @@ public String getTableType()
         return partitionColumns;
     }
 
+    public boolean isPartitioned()
```

Comment:
I recommend inlining this method. There are already quite a few places where we do `.getPartitionColumns().isEmpty` directly